pkg/hooks/manager.go

```diff
@@ -192,12 +192,12 @@ func (m *Manager) OnDefineDomain(domainSpec *virtwrapApi.DomainSpec, vmi *v1.Vir
 			if callback.Version == hooksV1alpha1.Version || callback.Version == hooksV1alpha2.Version {
 				vmiJSON, err := json.Marshal(vmi)
 				if err != nil {
-					return "", fmt.Errorf("Failed to marshal VMI spec: %v", vmi)
+					return "", fmt.Errorf("failed to marshal VMI spec: %v, err: %v", vmi, err)
 				}
 
 				conn, err := grpcutil.DialSocketWithTimeout(callback.SocketPath, 1)
 				if err != nil {
-					log.Log.Reason(err).Infof(dialSockErr, callback.SocketPath)
+					log.Log.Reason(err).Errorf(dialSockErr, callback.SocketPath)
```

Comment:
What would you say if we don't log anything in this function and simply "enhance" the errors we return. Then the caller's responsibility would be to log the reason(upon live migration we failed to call... or upon startup)

Reply:
Hm, not sure. If we would like to do that, I think we should also change that in `PreCloudInitIso` to make it consistent. Therefore, if we want to do it, I think we should do that in separate MR.

Reply:
But I like the idea in general, and I would be for it.

Reply:
Sure we can also adjust `PreCloudInitIso` but I would do it here as we have only this one log here for now. Otherwise, you would need to redo this and `PreCloudInitIso`.

Reply:
Yeah, and I would prefer that way. I don't want mix up too many things in one MR. Here I wanted to change whatever exists. Once merged, I will have another redo.